src/content/reference/hi/p5/normal.mdx

```diff
@@ -217,26 +185,25 @@ overloads:
     params:
       - name: vector
         description: |
-          <p>vertex normal as a <a href="#/p5.Vector">p5.Vector</a> object.</p>
+          <p>वर्टेक्स एक <a href="#/p5.Vector">p5.Vector</a> ऑब्जेक्ट के रूप में नार्मल है।</p>
         type: p5.Vector
     chainable: 1
   - line: 2241
     params:
       - name: x
         description: |
-          <p>x-component of the vertex normal.</p>
-        type: Number
-      - name: 'y'
+          <p>शीर्ष का x-घटक नार्मल।</p>
```

Comment:
This can be translated to clear sentences, do check the same in other files too

Reply:
Made the changes!